src/interface/home/Page.js

```diff
@@ -28,6 +28,7 @@ const Premium = lazyLoadComponent(() => retryingPromise(() => import(/* webpackC
 const About = lazyLoadComponent(() => retryingPromise(() => import(/* webpackChunkName: 'About' */ 'interface/about').then(exports => exports.default)));
 const HelpWanted = lazyLoadComponent(() => retryingPromise(() => import(/* webpackChunkName: 'HelpWanted' */ 'interface/helpwanted').then(exports => exports.default)));
 const ContributorPage = lazyLoadComponent(() => retryingPromise(() => import(/* webpackChunkName: 'ContributorPage' */ 'interface/contributor/Page').then(exports => exports.default)));
+const Search = lazyLoadComponent(() => retryingPromise(() => import(/* webpackChunkName: 'ContributorPage' */ 'interface/search').then(exports => exports.default)));
```

Comment:
The chunk name should differ from contributorpage.

Reply:
oh whoops I thought I changed that when I copied the previous line, guess I forgot. I'll fix that real quick